### PR TITLE
G3tSmurf Timing Updates

### PR DESCRIPTION
**Pull request by kmharrington**
This has been rebased off of #392  and #390  so we need to wait for these two to merge here. I would also like to wait until [this PR](https://github.com/simonsobs/smurf-streamer/pull/38) has started running at UCSD because I'd prefer to not need to repair the UCSD databases more than once (and this should technically be tested on "good" files). 

This PR adds a timing tracking column to the Files and Observation tables in G3tSmurf. It also adds our first example of the QDS monitor into the update-g3tsmurf-db script which will let us set up a grafana alter if the timing information dies at some point. 

Merging this PR will come with the need to run some table updates on the existing databases. I have scripts for these but @guanyilun  we'll need to coordinate running those scripts on the prefect managed databases. 

* Scripts to Update the databases / repair timing columns are here: https://github.com/simonsobs/pwg-scripts/tree/master/pwg-duf/g3tsmurf_timing 
* Monitor config files for this example are here: https://github.com/simonsobs/site-pipeline-configs

### CODE REVIEW

**Review by guanyilun — APPROVED** [review]
Looking good! You may consider also adding the `Monitor.from_configs` to the nice documentation here before we forget about this.

**Comment by kmharrington** [comment]
fixed the rebasing after merging the monitor PR (and checked that `from_configs` is caught by the autodoc). @guanyilun  go ahead and merge this when you have time to simultaneously do the database updates.

**Pull request merged**
